frontend/src/components/PumpAction.js

```diff
@@ -1,19 +1,24 @@
 import styled from "styled-components";
 import {useState} from "react";
-import {postMinutes} from "../service/timer-api-service";
+import {runPump, stopPump} from "../service/timer-api-service";
 
 export default function PumpAction(){
 
     const [minutes, setMinutes] = useState("")
 
-    const updateTimer = () => {
-        postMinutes(minutes).then(response => console.log(response))
+    const handleClickRunPumpButton = () => {
+        runPump(minutes).then()
+    }
+
+    const handleClickStopPumpButton = () => {
+        stopPump(minutes).then()
     }
 
     return(
         <Styled>
             <input type={"text"} onChange={event => setMinutes(event.target.value)} placeholder={"Minuten"}/>
```

Comment:
```suggestion
            <input type={"number"} onChange={event => setMinutes(event.target.value)} placeholder={"Minuten"}/>
```

Reply:
I think there's even an input time which you could configure to use minutes and seconds